db/migrate/20170909135441_add_title_and_verified_to_reviews.rb

```diff
@@ -0,0 +1,6 @@
+class AddTitleAndVerifiedToReviews < ActiveRecord::Migration[5.1]
+  def change
+    add_column :reviews, :title, :string
+    add_column :reviews, :verified, :boolean
```

Comment:
boolean field are likely to have `default: false, null: false` options

Reply:
fixed